### PR TITLE
chore(sql): improve alter table syntax error message

### PR DESCRIPTION
**Pull request by puzpuzpuz**

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|1|1|100.00%|

File: utils/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bluestreak01) [inline utils/dependency-reduced-pom.xml:1]: should remove

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: utils/dependency-reduced-pom.xml (hunk 1 above)
Comment (puzpuzpuz) [reply utils/dependency-reduced-pom.xml]: Done in 52ca1ebfed76073302306afa2b1f3d0aa6e80e80

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|1|1|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|1|1|100.00%|

**Pull request merged**